server/ansible/roles/pbench-repo-install/defaults/main.yml

```diff
@@ -1,8 +1,10 @@
 ---
-test_repo_enabled: 0
+fedoraproject_username: ndokos
+pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/{{ fedoraproject_username }}
 
 repos:
   - tag: pbench
+    user: "{{ fedoraproject_username }}"
     baseurl: "{{ pbench_repo_url_prefix }}/pbench/{{ distrodir }}"
     gpgkey: "{{ pbench_repo_url_prefix }}/pbench/pubkey.gpg"
     gpgcheck: 1
```

Comment:
Continuing with my previous comment, why is this being done under Nick's ID?  Shouldn't this be `pbench` or some similar project ID?  (What would have to be done to make that work?)  I wouldn't want to have to change this code if Nick's account were to go away for some reason or if someone else takes over his duties.